app/controllers/users_controller.rb

```diff
@@ -16,9 +17,10 @@ def create
   end
 
   def show
-    @user = User.find_by(params[:id])
-    @todos = @user.todos
-    @todo = Todo.new
+    result = ShowUser.call(id: params[:id])
```

Comment:
I like the consistency here. I agree that it might seem redundant to wrap a simple call in an interactor but it keeps everything the same. Also, it depends on what the action want's to do, maybe you can rename the interactor to `FindUser` instead so it's easier to understand what it's doing. 

The additional assignments below might not be necessary. If you need the user's todos you can do so by calling `@user.todos`. If you need a new user todo, you can also do that with `@user.todos.new`. The only assignment you really need is user, which you can accomplish in the same line of the interactor call by changing it to:

```ruby
@user = FindUser.call(id: params[:id]).user
```

The problem there though is that you still need to check for an error (what if the user was not found). ActiveRecord automatically handles `NotFound` errors but test that, to make sure we don't have to do anything else :smile:.